src/shared-components/button/style.ts

```diff
@@ -135,12 +135,6 @@ const disabledStyles = (theme: ThemeType) => `
   color: ${theme.COLORS.textDisabled};
   cursor: not-allowed;
   fill: ${theme.COLORS.textDisabled};
-
-  &:visited,
-  &:hover {
-    opacity: 1;
```

Comment:
So we no longer want to change the `opacity` and `color` on `visited` and `hover`?

Reply:
Ah never mind, this makes sense since those values weren’t actually being changed.

Reply:
Yep, and it seems like the redundant styling, paired with how we apply conditional styles, is part of what is causing the CI issues. 🤷